examples/worklet_message_port.rs

```diff
@@ -42,7 +42,7 @@ impl WhiteNoiseNode {
             output_channel_count: vec![1],
             parameter_data: HashMap::new(),
             processor_options: (),
-            channel_config: ChannelConfigOptions::default(),
+            channel_config: AudioNodeOptions::default(),
```

Comment:
Just a bit confused with the field name now, don't have a better idea though calling it `audio_node_options` would look a bit silly maybe...

Reply:
Very good point, since we use composition instead of inheritance all public `channel_config` keys should be renamed to `audio_context_options`